appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -1138,6 +1138,15 @@ Blockly.ReplMgr.processRetvals = function(responses) {
             rs.state = Blockly.ReplMgr.rsState.CONNECTED;
             Blockly.common.getMainWorkspace().fireChangeListener(new AI.Events.CompanionConnect());
             break;
+        case "startCache":
+            top.BlocklyPanel_startCache().then((success) => {
+                if (!success) {
+                    console.log("Failed to cache project.");
```

Comment:
We may want to propagate these errors up to the user if they happen, such as via a call to `runtimerr`.